docs/contributing/development-guide.md

```diff
@@ -174,22 +179,39 @@ force the branch to be in the same commit state as your local branch.
 
 ```
 
+### Typing using mypy
```

Comment:
@enadeau i added this so you can test running mypy locally across python versions now!! :)